Source/Core/Core/IOS/Network/KD/NWC24DL.h

```diff
@@ -44,11 +42,16 @@ class NWC24Dl final
   void SetVersion(u32 version);
 
 private:
-  enum
+  static constexpr u32 DL_LIST_MAGIC = 'WcDl';
```

Comment:
Looking this up, multi-char literals are apparently implementation-defined, so please convert this into the appropriate integer (in hexadecimal notation) and put the 'WcDl' in a comment.